### PR TITLE
Manually test user accepts payment request algo

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2017-09-12 05:21:52
Finished: 2017-09-12 05:28:29


<h2>Failing Jobs</h2>
<ul>

<li>chrome:unstable</li>

<li>safari:10.0</li>

<li>MicrosoftEdge:14.14393</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/18435)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/274465705)

**Comment by marcoscaceres** [comment]
@edenchuang, any chance you could take a look?

**Comment by rsolomakhin** [comment]
Can you add the overarching instructions? For example, "Click on each button in sequence from top to bottom without refreshing the page. Each button will bring up the PaymentRequest UI window."

**Comment by marcoscaceres** [comment]
@rsolomakhin good suggestion. I need to do that for all tests. I filed https://github.com/w3c/web-platform-tests/issues/7377 to do that.

File: payment-request/user-accepts-payment-request-algo-manual.https.html [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+        `response.${attr} must be ${expectedValue}`
+      );
+    }
+    await promise_rejects(
+      t,
+      "InvalidStateError",
+      request.show(),
+      "Request [[state]] is closed, so InvalidStateError"
+    );
+  }, testName);
```
Comment (edenchuang) [inline payment-request/user-accepts-payment-request-algo-manual.https.html]: using button.textContent.trim() directly and remove testName at line 82.

File: payment-request/user-accepts-payment-request-algo-manual.https.html [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+}
+
+</script>
+
+<section>
+  <h2 id="user-accepts-payment-request">User accepts payment request</h2>
+  <link rel="help" href="https://w3c.github.io/payment-request/#shipping-option-changed-algorithm">
```
Comment (edenchuang) [inline payment-request/user-accepts-payment-request-algo-manual.https.html]: should be https://w3c.github.io/payment-request/#user-accepts-the-payment-request-algorithm

File: payment-request/user-accepts-payment-request-algo-manual.https.html [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+        User accepts payment request, but not shipping is requested.
+      </button> Use any credit card to pay.
+    </li>
+    <li>
+      <button onclick="
+        const requestShipping = Object.assign({}, optionsRequestNothing, {requestShipping: true});
+        const expectedValues = { shippingOption: 'option 2' };
+        testAcceptRequestAlgorithm(this, detailsWithShippingOptions, requestShipping, expectedValues);">
+        User accepts payment request, merchant requests shipping.
+      </button> Select any shipping address, and use any credit card to pay.
```
Comment (edenchuang) [inline payment-request/user-accepts-payment-request-algo-manual.https.html]: I guess this should be shipping option not address.

File: payment-request/user-accepts-payment-request-algo-manual.https.html [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+        const requestPayerName = Object.assign({}, optionsRequestNothing, {requestPayerName: true});
+        const expectValues = { payerName: 'web platform test' };
+        testAcceptRequestAlgorithm(this, detailsNoShippingOptions, requestPayerName, expectValues);">
+        User accepts payment request, merchant requests payer's name.
+      </button>
+      When prompted, please use "web platform test" as the payer name.
+    </li>
+    <li>
+      <button onclick="
+        const requestPayerPhone = Object.assign({}, optionsRequestNothing, {requestPayerPhone: true});
```
Comment (edenchuang) [inline payment-request/user-accepts-payment-request-algo-manual.https.html]: Remove the redundant requestPayerPhone.

**Review by edenchuang — APPROVED** [review]
Couple of nits.

**Pull request merged**